src/modules/jcms_article_fragment/jcms_article_fragment.install

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @file
+ * Contains jcms_article_fragment install/update functions.
+ */
+
+/**
+ * Implements hook_update_N().
+ */
+function jcms_article_fragment_update_8101() {
+  \Drupal::entityDefinitionUpdateManager()->applyUpdates();
+}
```

Comment:
@nlisgo Having double checked this, this hook is necessary. The schema is not stored in configuration but in code so it needs to be updated this way.